backend/README.md

```diff
@@ -10,4 +10,37 @@ usando as bibliotecas abaixo:
 
 ## Gerenciamento dos serviços Google
 
-As informações estão contidas na documentação do [app aqui](../mobile/app).
\ No newline at end of file
+As informações estão contidas na documentação do [app aqui](../mobile/app).
+
+## Executando servidor localmente
+
+O servidor backend pode ser falcimente executado utilizando o **Docker Compose**. Caso ainda não tenha o **Docker Compose** instalado, basta seguir o tutorial disponível no neste [link](https://docs.docker.com/compose/install/). Além disso, é necessário que o usuário que irá executar o servidor, tenha direitos de execução de comandos do **Docker**, para isso, basta executar o comando abaixo (este é o processo utilizado para sistemas **linux**);
+
+```sh
+sudo usermod -aG docker `whoami`
+```
+
+Após a instalação e configuração é necessário reiniciar o computador para garantir que as alterações sejam funcionais.
+
+### Configurações adicionais
+
+O arquivo [docker-compose.yml](docker-compose.yml) carrega as configurações básicas do servidor de banco de dados e do servidor de aplicação (servidor *NodeJS*). Este arquivo executa o build dos arquivos ***Dockerfile***, tanto do banco de dados (localizado em [/database/Dockerfile](../database/Dockerfile)), quanto arquivo que se encontra neste diretório ([/backend/Dockerfile](../backend/Dockerfile)). Além disso o arquivo está configurado para executar o servidor de aplicação na **porta 10000**, e o servidor de banco de dados na **porta 10001**, caso seja necessário basta modificar estes números dentro do arquivo para ter estes servidores disponíveis em portas diferentes. Antes de executar o arquivo [docker-compose.yml](docker-compose.yml) ainda é necessário **adicionar a *secret key* do captcha**.
```

Comment:
Estamos usando arquivo .env` para definir todas as configurações do backend e mobile.
As portas, nome do banco, IP do banco, senha, secret key do captcha e outras configurações são todas definidas em tais arquivos.

O docker-compose.yml precisa usar tais arquivos no lugar de definir os valores estaticamente. 
Veja este [link](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option) de como usar arquivos .env no docker compose.

Você pode incluir os arquivos .env abaixo no seu repositório local. Eles são ignorados pelo .gitignore:

## ./backend/.env

```bash
#Não usar aspas nas variáveis do backend, pois a aplicação
#não está removendo estas aspas, mesmo usando o módulo dotenv.

#Usa porta externamente, quando executando o backend no Docker.
#Internamente é sempre porta 8080
PORT=8080
SERVER=IP_DO_SERVIDOR_NODE
DB_HOST=IP_DO_POSTGRES

DB_DATABASE=aglomerou
DB_USER=postgres
DB_PASSWORD=SENHA_DO_SEU_POSTTGRES
DB_PORT=5432
CAPTCHA_SECRET_KEY=
```

## mobile/app/.env

REACT_APP_GOOGLE_MAPS_API_KEY=